doc/manual/faq.rst

```diff
@@ -144,6 +144,15 @@ Experiment windows can be organized by using the following hotkeys:
 
 The windows will be organized in the order they were last interacted with.
 
+fix errors when restarting the management system after a crash?
+---------------------------------------------------------------
+
+On Windows in particular, abnormal shutdowns such as power outages or bluescreens can sometimes corrupt the organizational files used by the management system, resulting in errors to the tune of ``ValueError: source code string cannot contain null bytes`` when restarting. The easiest way to handle these problems is to delete the corrupted files and start from scratch.
+
+If the master itself fails to start, it may be necessary to delete the dataset database or even ``last_rid.pyon`` to start from scratch, but if the dashboard or browser fail, the problem is probably in the GUI configuration files, where the state of the GUI (arrangement of docks, applets, etc.) is kept. This file is backed up once whenever it is successfully loaded. Navigate to the user configuration directory (see :ref:`gui-config-files`) and look for a file suffixed ``.backup``. To restore the GUI, simply delete the corrupted configuration file and rename the backup to replace it.
+
+Similar problems can also result if any of the management system organizational files are edited manually, as manual edits can easily cause text encoding or formatting issues the system is not intended to have to handle. It is strongly discouraged to try editing any of these files manually. 
```

Comment:
Old forum posts seemed to indicate that 'people trying to edit these files in unsuitable editors' was a known issue at one point. Maybe because they're more accessible than the source depending on how you install. Otherwise if this isn't actually likely to happen I'd just remove the paragraph entirely.